scripts/ci/update-generator/main.go

```diff
@@ -140,32 +128,33 @@ func createInstaller(outputPath, channel, platform string) error {
 }
 
 func run() error {
+	var (
+		platform = fetchPlatform()
+		inDir    = defaultInputDir
+		outDir   = defaultOutputDir
+		branch   = constants.BranchName
+		version  = constants.Version
+	)
+
+	flag.StringVar(
+		&platform, "platform", platform,
+		"Target platform in the form OS-ARCH. Defaults to running os/arch or the combination of the environment variables GOOS and GOARCH if both are set.",
+	)
+	flag.StringVar(&inDir, "i", inDir, "Override directory to gather payload from.")
+	flag.StringVar(&outDir, "o", outDir, "Override directory to output archive to.")
+	flag.StringVar(&branch, "b", branch, "Override target branch. (Branch to receive update.)")
+	flag.StringVar(&version, "v", version, "Override version number for this update.")
```

Comment:
During debugging, I got lost following what might be going wrong with the global+init setup of flags. The above design matches that of payload-generator, and it's less noisy, so I hope it's acceptable.